Tests/Fixtures/Generated/DisableAll/Colors/literals-swift3-context-defaults-customname.swift

```diff
@@ -0,0 +1,21 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/SwiftGen/SwiftGen
```

Comment:
The moved files and these new dupes are not needed. Use the existing variations mechanism.